The configure script never actually checks for string.h so the
macro tested in ifdef is never defined. Other source files in
this software unconditionally include string.h so this patch
follows suit.

A more correct fix would be to adjust the autoconf input file
(configure.in) to check for string.h, and wrap the #include in
ifdef for the other 19 source files. The expedient option was
chosen since this software is long out of maintenance and
the patches would make no difference to building on OpenBSD.

$OpenBSD: patch-debugXML_c,v 1.1 2008/07/14 20:42:41 sthen Exp $
--- debugXML.c.orig	Mon Jul 14 11:39:13 2008
+++ debugXML.c	Mon Jul 14 11:39:58 2008
@@ -16,9 +16,7 @@
 #ifdef HAVE_STDLIB_H
 #include <stdlib.h>
 #endif
-#ifdef HAVE_STRING_H
 #include <string.h>
-#endif
 #include "xmlmemory.h"
 #include "tree.h"
 #include "parser.h"
